cli/src/main.rs

```diff
@@ -697,7 +670,7 @@ fn run<F: FieldElement>(
             // Already ran when computing bootloader inputs, nothing else to do.
         }
         (true, false) => {
-            let mut pipeline = pipeline_factory().with_prover_inputs(inputs);
+            let mut pipeline = pipeline.with_prover_inputs(inputs);
```

Comment:
Note that this can use the same pipeline as the one passed to `rust_continuations_dry_run` now, which is already advanced to the `AnalyzedAsm` stage. Before, it would run from the start again.